src/providers/MeetingProvider/MeetingManager.ts

```diff
@@ -277,18 +247,19 @@ export class MeetingManager implements AudioVideoObserver {
     let logger: ConsoleLogger | MultiLogger = consoleLogger;
 
     if (postLoggerConfig) {
-      const { name, batchSize, intervalMs, url, logLevel } = postLoggerConfig;
-      logger = new MultiLogger(
-        consoleLogger,
-        new MeetingSessionPOSTLogger(
-          name,
-          meetingSessionConfiguration,
-          batchSize,
-          intervalMs,
-          url,
-          logLevel
-        )
-      );
+      logger = new MultiLogger(consoleLogger);
+      // const { name, batchSize, intervalMs, url, logLevel } = postLoggerConfig;
```

Comment:
Assume this will be removed once merged with Venky's PR.

Reply:
Yes